utilities/routes.ts

```diff
@@ -39,10 +39,6 @@ export const initiatives = {
     name: 'Stanford Science Fellows',
     path: `${initiativesRoot}/stanford-science-fellows`,
   },
-  townCenter: {
-    name: 'Town Center',
-    path: `${initiativesRoot}/town-center`,
-  },
```

Comment:
We're removing the Town Center initiative. I have adjust the layout on the Initiative page accordingly